File: JSAlbadanI/Hola mundo.c
#include <stdio.h>

int main() {
    int n;
    printf("Ingrese el número de veces que desea imprimir 'hola mundo': ");
    scanf("%d", &n);
    
    for(int i = 0; i < n; i++) {
        printf("hola mundo\n");
    }
    
    return 0;
}

File: JSAlbadanI/Hola mundo.java
import java.util.Scanner;

public class HolaMundo {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingresa el número de veces que quieres imprimir 'Hola mundo': ");
        int n = scanner.nextInt();
        for (int i = 1; i <= n; i++) {
            System.out.println("Hola mundo");
        }
    }
}
